src/SearchRoom/SearchRoom.css

```diff
@@ -0,0 +1,48 @@
+:root {
+    --Image-size: 200px;
+    --title-font-size: 20px;
+    --text-font-size: 18px;
```

Comment:
`--Image-size` and `--text-font-size` are unused in this css file.
Can they be removed?